test/runtests.jl

```diff
@@ -3,12 +3,14 @@ using Test
 using Conda, PyCall
 using FITSIO, WCS
 
-ENV["PYTHON"]=""
+#ENV["PYTHON"]=""
```

Comment:
Why?

Reply:
I wasn't sure what to add here. Is there a need to specify a specific python version here?

Reply:
It's not clear to me why you wanted to change this line at all.  This is to force Conda's Python.